Sources/NetworkProtection/Diagnostics/NetworkProtectionLatencyMonitor.swift

```diff
@@ -21,7 +21,7 @@ import Network
 import Common
 import Combine
 
-final public class NetworkProtectionLatencyMonitor {
+public actor NetworkProtectionLatencyMonitor {
```

Comment:
Making this an actor already ensures that its methods are executed sequentially, so it doesn't make sense for this to be an actor and its methods and properties to be declared as `@MainActor`.

Why not do without `@MainActor` entirely?